include/zephyr/net/net_ip.h

```diff
@@ -385,9 +385,11 @@ struct cmsghdr {
 #undef NET_SOCKADDR_MAX_SIZE
```

Comment:
The commit message of the last commit `drivers: nsos: fix support of AF_PACKET` is a bit misleading as the change does not do any changes for the nsos driver, it just enables it to work.
Could you change the commit subject to `net: ip: Add check for packet socket for offloaded sockets`
Not sure what exactly to say in commit message. Could you describe there **why** this change is needed and also mention nsos driver there. Currently your commit message says it fixes things but does not describe why the fix works in this case (which is important to write here).

Reply:
Hmm, I missed your comment about merging the last commit. Yes, we could merge it in which case my comment is not really valid any more :smile: